CONTRIBUTING.md

```diff
@@ -35,20 +36,31 @@ rh-ai-kickstart/
 
 ## What are the repository requirements? 
 
+### Required
 
 * A descriptive repo name makes your kickstart easy to find and understand 
-* Include a description for your repository
-* `README.md` 
+* `README.md` - please use README structure found in [ai-kickstart-template](https://github.com/rh-ai-kickstart/ai-kickstart-template)
 * Put the images your README uses in the `assets/images/` folder
+* Be as descriptive as possible! REMEMBER: you aren't just documenting code, you're documenting the use case
+* Add a description to your repo's About section
 
-Here's why: 
+### Optional, but encouraged
+* Add "[tags](#tag-your-kickstart)" to the About section of your kickstart 
+
+## What criteria are we using? 
```

Comment:
Add something about how these criterias will be evaluated when comes time to promote the kickstart in various locations.